test/local/db.js

```diff
@@ -634,5 +721,111 @@ describe('redis enabled', () => {
       assert.equal(result, '{"frang":{}}')
     })
   })
+
+  describe('mock db.pruneSessionTokens:', () => {
+    beforeEach(() => {
+      db.pruneSessionTokens = sinon.spy(() => P.resolve())
+    })
+
+    describe('return expired tokens from pool.get:', () => {
+      beforeEach(() => {
+        const expiryPoint = Date.now() - tokenLifetimes.sessionTokenWithoutDevice
+        pool.get = sinon.spy(() => P.resolve([
+          { tokenId: 'unexpired', createdAt: expiryPoint + 1000 },
+          { tokenId: 'expired1', createdAt: expiryPoint - 1 },
+          { tokenId: 'expired2', createdAt: 1 }
+        ]))
+      })
+
+      it('should call pruneSessionTokens in db.sessions', () => {
```

Comment:
I couldn't think of a way to assert this from the remote tests (because `createdAt`, so we only have unit-test coverage for the call from `db.sessions`. `db.pruneSessionTokens` itself is covered by remote tests though.